examples/angularjs/node_modules/angular/angular.js

```diff
@@ -1,9 +1,9 @@
 /**
- * @license AngularJS v1.4.3
- * (c) 2010-2015 Google, Inc. http://angularjs.org
+ * @license AngularJS v1.5.5
```

Comment:
I suspect this is what broke the CI tests. There is a backward-incompatible change in AngularJS 1.5.x, which breaks this example. See angular/angular.js#15642.